### PR TITLE
Fix slight typo in CONTRIBUTING.md.

### PR DESCRIPTION
**Pull request by EthanJoyce**
"Then right in ..." should be "Then right click ..."

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-# Contributing
+#
```
Comment (raelgc) [inline CONTRIBUTING.md:1]: @mrlolethan Why this break line here? It breaks the title

Comment (EthanJoyce) [inline CONTRIBUTING.md:1]: @raelgc I have no idea; the GitHub in-browser editor did that. Shall I make another commit to fix that, or?

Comment (raelgc) [inline CONTRIBUTING.md:1]: @mrlolethan Well, the important here is you alerted me about the typo. I'll edit the file and fix it. And yeah, sometimes the in-browser editor break lines by itself :(

Comment (EthanJoyce) [inline CONTRIBUTING.md:1]: @raelgc All right.
